include.py

```diff
@@ -59,9 +59,9 @@ def get(self):
         self.r=[]
         try:
             # If project already exists in simulations table (e.g. adding 100 simulations to existing 1000)
-            _max=self.p.query("SELECT max(iteration)+1 FROM simulations WHERE project={} AND scenario_id={}".format(self.project,self.scenario_id))[0][0]
-            self.r.append(_max-self.how_many)
+            _max=self.p.query("SELECT max(iteration)+1 FROM simulations WHERE project={} AND scenario_id={} AND host IS NOT NULL".format(self.project,self.scenario_id))[0][0]
```

Comment:
Could you elaborate on this? Maybe we can chat on Teams.

Reply:
There is a difference between SimIterations and include.create_sequence()

Now those are unified.